README.md

```diff
@@ -29,10 +29,45 @@ Our idea is build a telegram bot for community management, it has 3 functionalit
 2. ton-connect: Used to gather user's information
 3. telegram robot: Used to interact with users
 
-# API Documentation
-- [Link to documentation](https://github.com/LeadBest/ton-hackthon/blob/master/dooby-api/README.md)
+## Deployment Steps
+
+### 1. Deploy NFT Collection Contract
+```
+cd ton-nft-deployer
+npm install
+npm run start
+```
+### 2. Run local API Server
+```
+cd dooby-api
+```shell
+# 1. build image
+$ docker build -t dooby-app .
+
+# 2. test at local
+$ docker run -e PORT=80 -p 80:80 dooby-app
+```
+### 3. Run local Web Server
+```
+cd webview
+npm install
+npm start
+```
+### 4. Put nftCloudFunction into Google Cloud function endpoint
+```
+cd nftCloudFunction
+npm install
+npm run start
+```
+### 5. Put appScript into Google appScipt endpoint
+1. Create 4 new project in google appScript
+2. Paste the 4 `.gs` files individually into each project
+#### Reference
+[Google Apps Script](https://developers.google.com/apps-script)
+[Google appScript Console](https://script.google.com/home/start)
 
 ## Reference
 
 [Idea Agenda](https://docs.google.com/presentation/d/1QEUo_2glJO5DsWRdGZ5p6H0i_8olk6lu865NMJp1fm0/edit#slide=id.g2052594a4f0_2_63)
```

Comment:
Repeat. Same link for **Demo slide**

Reply:
Removed

Reply:
Removed, plz resolve it

Reply:
https://github.com/LeadBest/ton-hackthon/tree/README_update